app/views/registration/organisation_details.scala.html

```diff
@@ -15,19 +16,31 @@
 
     <form method="post">
         @CSRF.formField
-        <div class="form-group">
-            @input(form("companyName"), '_label -> "Company name", '_inputClass -> "form-control form-control--block")
-        </div>
-        <div class="form-group">
-            @input(form("ctrNumber"), '_label -> utrLabel, '_inputClass -> "form-control form-control--block")
-        </div>
-        <div class="form-field form-field--submit">
-            <input type="submit" class="button" value="Continue" id="continue">
-        </div>
-        <div class="form-field form-field--submit">
-            <p><a href="javascript:history.go(-1)" class="back-link" id="back">Back</a></p>
-        </div>
+
+            <div class="form-group">
+                @input(form("companyName"), '_label -> "Company name", '_inputClass -> "form-control form-control--block")
+            </div>
+            <div class="form-group">
+                @input(form("ctrNumber"),
+                '_label -> utrLabel,
+                '_inputHint -> "For example, if your Tax reference number is '1234567890123' enter the last 10 digits.",
+                '_hintId -> "hint-utr-text",
+                '_inputClass -> "form-control form-control--block")
+            </div>
+            <div class="form-field form-field--submit">
+                <input type="submit" class="button" value="Continue" id="continue">
+            </div>
+            <div class="form-field form-field--submit">
+                <p><a href="javascript:history.go(-1)" class="back-link" id="back">Back</a></p>
+            </div>
+
     </form>
 
+    <style type="text/css">
```

Comment:
Not happy merging the code until we can put this in a stylesheet somewhere but I can sort that out if need be.